src/txorphanpool.h

```diff
@@ -18,11 +18,11 @@
 class CTxOrphanPool
 {
 private:
-    // Used in EraseOrphansByTime() to track when the last time was we checked the cache for anything to delete
+    //! Used in EraseOrphansByTime() to track when the last time was we checked the cache for anything to delete
```

Comment:
why adding the exclamation mark in the beginning?

Reply:
no reason, i'll change it back..it's used elsewhere in the codebase but not really our standar..